etc/samples/logging.yaml.journald.sample

```diff
@@ -0,0 +1,37 @@
+version: 1
+disable_existing_loggers: False
+formatters:
+    simple:
+        format: "%(asctime)s - %(name)s - %(levelname)s - %(message)s"
+        datefmt: "%Y-%m-%dT%H:%M:%S%z"
+
+handlers:
```

Comment:
Would use something like /var/log/tendrl/bridge_common_info.log and /var/log/bridge_common_errors.log